<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GENOVA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="buttonsFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.041, y1:0, x2:0.989, y2:0.0113636, stop:0 rgba(5, 0, 167, 255), stop:0.982955 rgba(25, 86, 255, 255), stop:1 rgba(5, 9, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>341</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QToolButton" name="page1Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Birth</string>
      </property>
      <property name="icon">
       <iconset resource="gene.qrc">
        <normaloff>:/Pictures/icons/GenovaBIRTH_2.svg</normaloff>:/Pictures/icons/GenovaBIRTH_2.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="page2Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Life</string>
      </property>
      <property name="icon">
       <iconset resource="gene.qrc">
        <normaloff>:/Pictures/icons/GenovaLIFE.svg</normaloff>:/Pictures/icons/GenovaLIFE.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="page3Button">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Death</string>
      </property>
      <property name="icon">
       <iconset resource="gene.qrc">
        <normaloff>:/Pictures/icons/GenovaDEATH.svg</normaloff>:/Pictures/icons/GenovaDEATH.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="page4Button">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Absolute</string>
      </property>
      <property name="icon">
       <iconset resource="gene.qrc">
        <normaloff>:/Pictures/icons/GenovaABSOLUTE.svg</normaloff>:/Pictures/icons/GenovaABSOLUTE.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>731</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>47</width>
        <height>13</height>
       </size>
      </property>
      <property name="text">
       <string>Genome Length</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Genome Symbols</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Population Size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Generations</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0,1</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="genome_spinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="population_spinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="generations_spinBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string>Infinity</string>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="chromosome_spinBox">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>516</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Chromosome Length</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chromosome_checkBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Fixed</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="genome_checkBox">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>40</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fixed</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="population_checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fixed</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Scoring</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Recombination</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Clone Elites</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Random</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="cloning_spinBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileSelectLineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="fileSelectButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mutation</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="mutation_spinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Resurrection</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>By Mutation</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>By Splicing</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="resurrection_spinBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Culling</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fitness Threshold</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="culling_spinBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>disable splicing for fixed length genomes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>disable culling for fixed population sizes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="resultTextEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>-10</y>
        <width>271</width>
        <height>311</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGENOVA">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuGENOVA"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gene.qrc"/>
 </resources>
 <connections/>
</ui>
